auth.proto

```diff
@@ -32,13 +32,16 @@ message VerifyTxReq {
 }
 
 message VerifyBlockReq {
-    uint64 id = 1;
-    repeated VerifyTxReq reqs = 2;
+    uint64 height = 1;
+    uint64 round = 2;
+    CompactBlock block = 3;
 }
 
 message VerifyBlockResp {
-    uint64 id = 1;
-    Ret ret = 2;
+    uint64 height = 1;
+    uint64 round = 2;
+    Ret ret = 3;
```

Comment:
`bool` is ok.

Reply:
Yes. I delete `Ret` and add `bool pass` as the flag if the verification is passed or not.